content/docs/concepts/replication/replication-opvirt.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Metro Replication with OpenShift Virtualization
+linktitle: Metro Replication with OpenShift Virtualization
+weight: 6
+description: >
+  Support of Metro Replication with OpenShift Virtualization for CSI drivers.
+---
+
+Starting in v1.12.0, the Replication module supports the metro volume
+replication for Persistent Volumes (PVs) created via virtualized workloads
+(VMs).
+
+## OpenShift Virtualization:
+
+- OpenShift Virtualization allows virtual machines (VMs) to run alongside
+  container workloads within Kubernetes clusters, addressing the coexistence of
+  legacy VM workloads and modern containers.
+
+## OpenShift Virtualization support with Metro Replication
+
+- Install the OpenShift Virtualization operator from the RedHat UI on OpenShift
+  Cluster.
+- The Replication modes supported are: Metro

```

Comment:
Can this be in bold? I want to make sure customer do not enable other mode and expected to work

Reply:
Done.